example/pubspec.yaml

```diff
@@ -0,0 +1,8 @@
+name: httpexample
```

Comment:
This isn't necessary, right? Just drop the pubspec. The package site picks this up fine.

Reply:
How would you run the example then?

Reply:
`dart example/example.dart` 🤷‍♂️

Reply:
```
$ ls -a
.		..		main.dart
$
```

Reply:
Never mind, my mistake, I had moved this out into another directory structure. Working now...